ROOTFS/etc/waggle/sanity/fatal/network_switch.test

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+repeat() {
+    n=$1
+    shift
+    for _ in $(seq $n); do
+        $*
+        sleep 0.1
+    done
+}
+
+if ! dev=$(ls /dev/serial/by-id/usb-FTDI* | head -n 1); then
+    echo "Network Switch Test: no serial console found FAIL"
+    exit 1
+fi
+
+# setup serial so we can interact with standard io tools
+stty -F $dev 115200 brkint -icrnl -imaxbel -opost -onlcr -isig -icanon -echo min 100 time 2
+
+# clear any buffered data
+timeout 3 cat "$dev" > /dev/null
```

Comment:
just checking that that if you experience a timeout here (`timeout` returns an error) you want to keep running.  This is just a flush right?

Reply:
That's right - this is intended to flush any data. (Without the timeout, cat reads forever.)